packages/libs/pactjs-test-project/src/example-templates/generated-tx-templates/index.ts

```diff
@@ -11,17 +11,17 @@ export default {
     fromKey: string;
   }): ICommandBuilder<{}> => {
     const parts = [
-      'code: |-\r\n  (coin.transfer "',
+      'code: |-\n  (coin.transfer "',
       '" "',
       '" ',
-      ')\r\ndata:\r\npublicMeta:\r\n  chainId: "',
-      '"\r\n  sender: ',
-      '\r\n  gasLimit: 2500\r\n  gasPrice: 1.0e-8\r\n  ttl: 600\r\nnetworkId: ',
-      '\r\nsigners:\r\n  - pubKey: ',
-      '\r\n    caps:\r\n      - name: "coin.TRANSFER"\r\n        args: [',
```

Comment:
Should we use snapshot testing here to prevent issues with line endings? Similar to https://github.com/kadena-community/kadena.js/blob/8b5603f453722e50751c95c96fc483f1a2efa880/packages/libs/pactjs-generator/src/template/generation/tests/generateTemplates.test.ts#L51

Reply:
My bad, I though this was a test. Leave it for now. We might want to remove this one from the repo